lib/widgets/poll.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/zulip_localizations.dart';
+
+import '../api/model/submessage.dart';
+import 'content.dart';
+import 'store.dart';
+import 'text.dart';
+
+class PollWidget extends StatelessWidget {
+  const PollWidget({super.key, required this.poll});
+
+  final Poll poll;
+
+  @override
+  Widget build(BuildContext context) {
+    final zulipLocalizations = ZulipLocalizations.of(context);
+    final store = PerAccountStoreWidget.of(context);
+    final theme = ContentTheme.of(context);
+
+    final textStylePollBold = const TextStyle(fontSize: 18)
+      .merge(weightVariableTextStyle(context, wght: 600));
+    final textStylePollNames = TextStyle(
+      fontSize: 16, color: theme.colorPollNames);
+
+    String getOptionVoterNames(PollOption option) => option.voters.map((userId) =>
+      store.users[userId]?.fullName
+        ?? zulipLocalizations.unknownUserName).join(', ');
```

Comment:
Let's add a `TODO(i18n)` like this:

```dart
        // TODO(i18n): List formatting, like you can do in JavaScript:
        //   new Intl.ListFormat('ja').format(['Chris', 'Greg', 'Alya', 'Zixuan'])
        //   // 'Chris、Greg、Alya、Zixuan'
```

(we have this TODO in a few places already)